src/plugins/vis_augmenter/public/utils/utils.ts

```diff
@@ -38,7 +39,10 @@ export const isEligibleForVisLayers = (vis: Vis): boolean => {
   const hasOnlyLineSeries =
     vis.params.seriesParams.every((seriesParam: { type: string }) => seriesParam.type === 'line') &&
     vis.params.type === 'line';
-  return hasValidXaxis && hasCorrectAggregationCount && hasOnlyLineSeries;
+  // Checks if the augmentation setting is enabled
+  const config = getUISettings();
+  const isAugmentationEnabled = config.get('visualization:enablePluginAugmentation');
```

Comment:
I imagine all of these checks will need an additional one to check the yml setting right?

Reply:
When the setting is undefined, it is evaluated as false and works as expected